includes/compatibility.php

```diff
@@ -229,37 +231,71 @@ public static function filter_library_post_type( $post_types ) {
 	}
 
 	/**
-	 * Polylang compatibility.
+	 * Set editor language.
 	 *
-	 * Fix Polylang compatibility with Elementor.
-	 *
-	 * @since 2.0.0
+	 * @since 3.7.0
 	 * @access private
 	 * @static
 	 */
-	private static function polylang_compatibility() {
-		// Fix language if the `get_user_locale` is difference from the `get_locale
+	private static function set_editor_language() {
 		if ( isset( $_REQUEST['action'] ) && 0 === strpos( $_REQUEST['action'], 'elementor' ) ) {
 			add_action( 'set_current_user', function() {
 				global $current_user;
-				$current_user->locale = get_locale();
-			} );
 
-			// Fix for Polylang
-			define( 'PLL_AJAX_ON_FRONT', true );
+				$elementor_preferences = get_user_meta( $current_user->ID, 'elementor_preferences' );
```

Comment:
I tried to use this code instead, but it always return null
`$user_interface = SettingsManager::get_settings_managers( ‘editorPreferences’ )->get_model()->get_settings( ‘user_interface’ );`